sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/pkversion/PartitionControllerImpl.java

```diff
@@ -168,7 +168,17 @@ private Mono<Void> getWorkerJob(
     }
 
     private Mono<Void> handleSplit(Lease lease, String lastContinuationToken) {
-        lease.setContinuationToken(lastContinuationToken);
+        if (lastContinuationToken != null) {
+            logger.warn("Partition {}, with owner: {}, updated with last continuation token: {}",
```

Comment:
```suggestion
            logger.info("Partition {}, with owner: {}, updated with last continuation token: {}",
```

Reply:
discussed offline, will keep it warn level